src/middlewared/middlewared/plugins/webui/crypto.py

```diff
@@ -26,9 +29,13 @@ class Config:
     async def certificate_profiles(self):
         return CERTPROFILES
 
-    @accepts(roles=['READONLY_ADMIN'])
+    @api_method(
+        CAProfilesArgs,
+        CAProfilesResult,
+        roles=['READONLY_ADMIN']
+    )
```

Comment:
We have CERTIFICATE_AUTHORITY_READ role

Reply:
I think we should use explicit crypto-related read roles in this plugin.

Reply:
I'll do so in another PR

Reply:
meh, I went ahead and fixed while I was here.